core/trino-main/src/test/java/io/trino/security/TestFileBasedSystemAccessControl.java

```diff
@@ -15,6 +15,7 @@
 
 import com.google.common.collect.ImmutableMap;
```

Comment:
Can we rename the commit message as `Extract new module for FileBasedSystemAccessControl`

Reply:
done